### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?

As the title says 
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,60 @@
...
+
+This app has an interface that describes what are the rules that the blocks must implement when you want to use the telemarketing block. 
+
+```json
+{
+  "telemarketing": {
+    "component": "index"
+  }
+}
+```
```
Comment (brunojdo) [inline README.md]: Where's the configuration section?

Comment (vitoriaHeliane) [reply README.md]: This app does not have configuration.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,61 @@
...
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-To import it into your code: 
-```js
-import Telemarketing from 'vtex.telemarketing/index'
-```
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
```
Comment (GustavoMA) [inline README.md]: You don't have any `API` header. Delete this from the Table of Contents.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,60 @@
...
-You can use it in your code like a React component with the jsx tag: `<Telemarketing />`. 
-```html
-<Telemarketing />
+
+This app uses our store builder with the blocks architecture. To know more about the Store Builder [click here](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object).
+
+To use this app you need to import it in your dependencies on `manifest.json` file.
+
+```json
+  dependencies: {
```
Comment (GustavoMA) [inline README.md]: `"dependencies"`

File: README.md [hunk 4]
```diff
@@ -1,27 +1,64 @@
...
+```
+
+#### Configuration 
+Configuration NDA.
+
+### Styles API
+:construction: :construction: :construction:
+
+## Troubleshooting
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/telemarketing/issues). Also feel free to [open issues](https://github.com/vtex-apps/telemarketing/issues/new) or contribute with pull requests.
```
Comment (GustavoMA) [inline README.md]: Missing Tests, see minicart readme

File: README.md [hunk 5]
```diff
@@ -1,27 +1,98 @@
...
+| `logout`     | [LogoutCustomerSession](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LogoutCustomerSession.tsx)   | Container of the logout     |
+| `logoutForm`    | [LogoutCustomerSession](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LogoutCustomerSession.tsx)   | Container of the logout form                     |
+| `popoverHeaderIcon`      | [LoginAsCustomer](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LoginAsCustomer.tsx), [LogoutCustomerSession](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LogoutCustomerSession.tsx)          | Container of the icon that appear in the popover header                      |
+| `popoverHeaderEmail`          | [LoginAsCustomer](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LoginAsCustomer.tsx), [LogoutCustomerSession](https://github.com/vtex-apps/telemarketing/blob/master/react/components/LogoutCustomerSession.tsx)  | Container of the email that appear in the popover header                            |
+
+To override the default CSS, you need to import `styles` on your manifest:
```
Comment (brunojdo) [inline README.md]: Move this section before de explanation of the table.

File: README.md [hunk 6]
```diff
@@ -1,27 +1,113 @@
...
+To configure or customize this app, you need to import it in your dependencies in `manifest.json`.
+
+```json
+  dependencies: {
+    "vtex.telemarketing": "2.x"
+  }
 ```
 
-DISCLAIMER: This component will only be displayed if the user has the role of `2 - Televendas (Call center operator)` in it´s profile access.
+Then, add `telemarketing` block into your app theme as we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json).
```
Comment (GustavoMA) [inline README.md]: Now, you can change the behavior of the telemarketing block that is in the store header. See an example of how to configure:
**example**

Comment (GustavoMA) [reply README.md]: You need to add this, see minicart's example.

File: README.md [hunk 7]
```diff
@@ -1,27 +1,113 @@
...
+}
+```
+
+#### CSS namespaces
+
+Below, we describe the namespaces that are defined in the telemarketing.
+
+| Class name         | Description          | Component Source                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | The main container of telemarketing| [index](https://github.com/vtex-apps/telemarketing/blob/master/react/Telemarketing.tsx)                                  |
```
Comment (GustavoMA) [inline README.md]:
```suggestion
| `container`        | The main container of telemarketing| [index](/react/Telemarketing.tsx)                                  |
```

Comment (GustavoMA) [reply README.md]: You can shorten all the links like this.

File: README.md [hunk 8]
```diff
@@ -1,27 +1,113 @@
...
+  "telemarketing": {
+    "component": "index"
+  }
+}
+```
+
+As you can see, this app has no required or optional block.
+
+#### Configuration 
+Configuration NDA.
```
Comment (GustavoMA) [inline README.md]: Is there any configuration in this app? Isn't better to say that it does not have configurations props for now?

File: CHANGELOG.md [hunk 9]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (GustavoMA) [inline CHANGELOG.md]: @lucasecdb recomendation: does not add this to changelog.

File: README.md (hunk 6 above)
Comment (vitoriaHeliane) [reply README.md]: This app has no behavior to change. So I think it's not necessary to add an example with an empty json.

**Pull request merged**